mappings/net/minecraft/recipe/RecipeGridAligner.mapping

```diff
@@ -6,3 +6,9 @@ CLASS net/minecraft/unmapped/C_wuyfqgmj net/minecraft/recipe/RecipeGridAligner
 		ARG 4 recipe
 		ARG 5 inputs
 		ARG 6 amount
+	METHOD m_vnshednj collectToInput (Ljava/lang/Object;IIII)V
+		ARG 1 itemId
+		ARG 2 iSlot
```

Comment:
why not `slotIndex`?

Reply:
Oh, that's something I do in my own code because it's brief and works when there are multiple indices (`jSlot`), forgot it's not standard for QMap.

I'll change it.